### PR TITLE
ruskakimov/track_only_successful_email_submits

### PR DESCRIPTION
**Pull request by ruskakimov**

### CODE REVIEW

File: src/javascript/static/pages/home.js [hunk 1]
```diff
@@ -60,9 +62,21 @@ const Home = (() => {
...
+            });
         } else {
             $('.signup-box div').replaceWith($('<p/>', { text: localize('Thank you for signing up! Please check your email to complete the registration process.'), class: 'gr-10 gr-centered center-text' }));
             $('#social-signup').setVisibility(0);
+            GTM.pushDataLayer({
+                event                   : 'email_submit',
+                email_submit_input      : response.echo_req.verify_email,
+                email_submit_days_passed: moment().diff(moment(localStorage.getItem('date_first_contact')), 'days'),
+                email_submit_source     : 'binary.com',
+            });
```
Comment (negarn) [inline src/javascript/static/pages/home.js]: why not just declare the common variable and push the email submit source to it later then send it once?

Comment (ruskakimov) [reply src/javascript/static/pages/home.js]: if we never get an error then sure 👍

Comment (negarn) [reply src/javascript/static/pages/home.js]: it can be in a bigger else wrapper

File: src/javascript/static/pages/home.js [hunk 2]
```diff
@@ -55,15 +57,22 @@ const Home = (() => {
...
+        if (isBinaryApp()) {
             BinaryPjax.load(urlFor('new_account/virtualws'));
         } else {
             $('.signup-box div').replaceWith($('<p/>', { text: localize('Thank you for signing up! Please check your email to complete the registration process.'), class: 'gr-10 gr-centered center-text' }));
             $('#social-signup').setVisibility(0);
         }
+        GTM.pushDataLayer({
+            event                   : 'email_submit',
+            email_submit_input      : response.echo_req.verify_email,
+            email_submit_days_passed: moment().diff(moment(localStorage.getItem('date_first_contact')), 'days'),
```
Comment (negarn) [inline src/javascript/static/pages/home.js]: do we need to convert to utc?

Comment (ruskakimov) [reply src/javascript/static/pages/home.js]: good point, `date_first_contact` is in UTC

**Pull request merged**